base/compiler/typeinfer.jl

```diff
@@ -453,8 +453,15 @@ function resolve_call_cycle!(linfo::MethodInstance, parent::InferenceState)
     return false
 end
 
+const inference_graph = IdDict{Any,IdSet{Any}}()
+const record_edges = Ref(false)
```

Comment:
```suggestion
const record_edges = Bool[false]
```

maybe (for bootstrap)

Reply:
Is `Ref` not defined early enough? Maybe a comment to that effect since that was the first thing I wondered when seeing the current state of the PR.

Reply:
`Ref` is available, but no constructors. Use `RefValue`